fk/status.go

```diff
@@ -85,6 +85,8 @@ func statusSubcommand(args docopt.Opts) exitCode {
 				colour.Cmd("fk team join " + groupedMembership.Team.UUID.String()),
 			}))
 		}
+
+		out.Print("To fetch and store team members keys run " + colour.Cmd("fk team fetch") + "\n\n")
```

Comment:
How about "The team's keys are updated automatically. To check for updates now, run fk team fetch"

Reply:
Sounds good to me!